app/mailers/late_reservation_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class LateReservationMailer < ApplicationMailer
+  def forgot_to_buy_gift_email(user)
+    @user = user
+    @url  = 'http://example.com/login'
```

Comment:
yeah, but you don't need parentheses after a method call if it takes no arguments